docs/sources/examples/mongodb.md

```diff
@@ -133,19 +136,19 @@ as daemon process(es).
 
     # Basic way
     # Usage: docker run --name <name for container> -d <user-name>/<repository>
-    $ docker run --name mongo_instance_001 -d my/repo
+    $ docker run -p 27017:27017 --name mongo_instance_001 -d my/repo
```

Comment:
> However, that's no different from running multiple MongoDB process on the same machine; each would need its own port number.

You're right; some users blindly copy the examples, and in this case, running it multiple times won't work :smile:. 

I'm okay with using the fixed port number, some things to consider;
- The "read the port from docker ps" doesn't really make sense in this case (fixed port)
- Perhaps *show* how to launch multiple instances, or (the reverse) remove the part about "one or more instances"